llvm/test/CodeGen/NVPTX/store-undef.ll

```diff
@@ -108,6 +108,40 @@ define void @test_store_volatile_undef(ptr %out, <8 x i32> %vec) {
 ; CHECK-NEXT:    st.volatile.v4.u32 [%rd4], {%r7, %r8, %r9, %r10};
 ; CHECK-NEXT:    ret;
   store volatile %struct.T undef, ptr %out
-  store volatile <8 x i32> %vec, <8 x i32>* poison
+  store volatile <8 x i32> %outVec, ptr undef
```

Comment:
This is not testing the store of undef vector case. The vector is more relevant than the aggregate, which doesn't survive the DAG builder